arch/arm64/boot/dts/amlogic/mesongxm-gpu-t82x.dtsi

```diff
@@ -32,7 +32,7 @@
             125000 1000000
         >;
 
-        tbl = <&dvfs125_cfg &dvfs285_cfg &dvfs400_cfg &dvfs500_cfg &dvfs666_cfg>;
+        tbl = <&dvfs285_cfg &dvfs400_cfg &dvfs500_cfg &dvfs666_cfg &dvfs800_cfg &dvfs800_cfg>;
 
```

Comment:
Perhaps operating-points table above also needs to be changed? Also why dvfs125_cfg has been removed and there are two dvfs800_cfg values in tbl?

Reply:
Thank you for your review.

`operating-points` table is not used anywhere for Mali, although I adjusted it for clarity.

If you look at the lines I deleted from gxm trees, `dvfs125_cfg` was not used for any board, perhaps because of stability/performance issues? I kept it that way. Double `dvfs800_cfg` is needed to allow the driver to scale to max frequency, as you can see `dvfs666_cfg` was repeated previously. I tested with single `dvfs800_cfg` and it didn't work.